lib/movie_genres.dart

```diff
@@ -0,0 +1,53 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_project/styles.dart';
+
+
+class MovieGenres extends StatelessWidget {
+  const MovieGenres({
+    super.key,
+    required this.movieGenres,
+  });
+
+  final List movieGenres;
+
+  static const double paddingBottomGenres = 15;
+  static const double genreFontSize = 15;
+  static const double borderRadiusGenreCard = 20;
+  static const double paddingGenreSpace = 12;
+
+  static const TextStyle genreText = TextStyle(
+    fontSize: genreFontSize,
+    color: Colors.white54,
+  );
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.only(
+        bottom: paddingBottomGenres,
+      ),
+      child: SingleChildScrollView(
+        scrollDirection: Axis.horizontal,
+        child: Row(
```

Comment:
This could be replaced with a horizontal `ListView` widget.
https://zaidrazanarsinh.hashnode.dev/choosing-the-right-scrolling-widget-in-flutter-listview-vs-singlechildscrollview

Reply:
We'll leave it as it is, to support dynamic children sizing.